vignettes/articles/00-prerequisites.Rmd

```diff
@@ -44,16 +41,16 @@ Helpful links for understanding PACTA for Banks and guidance material to help ru
 
 * PACTA for Banks overview:
 https://www.transitionmonitor.com/pacta-for-banks-2020/
-* PACTA for Banks MethodologyÖ https://www.transitionmonitor.com/wp-content/uploads/2021/07/PACTA-for-Banks-Methodology-document-02-07-2021_v1.2.0_v4.pdf
-* The PACTA for Banks R packages can be obtained from CRAN or GitHub. The material below describes in step by step how to use them
-r2dii.data manual: https://2degreesinvesting.github.io/r2dii.data/
-r2dii.match manual: https://2degreesinvesting.github.io/r2dii.match/
-r2dii.analysis manual: https://2degreesinvesting.github.io/r2dii.analysis/
+* PACTA for Banks Methodology: https://www.transitionmonitor.com/wp-content/uploads/2021/07/PACTA-for-Banks-Methodology-document-02-07-2021_v1.2.0_v4.pdf
+* The PACTA for Banks R packages can be obtained from CRAN or GitHub. The material below describes in step by step how to use them:
+  + r2dii.data manual: https://2degreesinvesting.github.io/r2dii.data/
+  + r2dii.match manual: https://2degreesinvesting.github.io/r2dii.match/
+  + r2dii.analysis manual: https://2degreesinvesting.github.io/r2dii.analysis/
 
 ## Next steps
 
 After successfully completing the prerequisites, see the Readme on how to install the packege followed by the below-mentioned vignettes on how to run the analysis:
 
-* ["Readme with installation procedure"](https://2degreesinvesting.github.io/r2dii.climate.stress.test/index.html)
-* ["Set up project directories for climate transition risk stress test"](https://2degreesinvesting.github.io/r2dii.climate.stress.test/articles/articles/02-setup-project-directories.html)
-* ["Preparing the intermediate inputs for stress testing loans"](https://2degreesinvesting.github.io/r2dii.climate.stress.test/articles/articles/03-prepare-loans-inputs.html)
+* [Readme file with installation procedure](https://2degreesinvesting.github.io/r2dii.climate.stress.test/index.html)
```

Comment:
Users cannot download this link as a file... maybe "Readme document with installation procedure"?